src/store/audio.ts

```diff
@@ -614,37 +587,47 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
         state,
         voice,
         presetKeyCandidate,
-        state.inheritAudioInfo && baseAudioItem !== undefined
+        baseAudioItem ? "copy" : "generate"
       );
-      audioItem.presetKey = nextPresetKey;
+      newAudioItem.presetKey = nextPresetKey;
+
+      // audioItemに対してプリセットを適用する
+      if (shouldApplyPreset) {
+        if (nextPresetKey) {
+          const preset = state.presetItems[nextPresetKey];
+          const result = applyAudioPresetToAudioItem(newAudioItem, preset);
+
+          if (result) {
+            return result;
+          }
+        }
+      }
 
+      // プリセットを適用しないならパラメータを引き継ぐ
       if (
         state.inheritAudioInfo &&
         baseAudioItem &&
         baseAudioItem.query &&
-        audioItem.query
+        newAudioItem.query
       ) {
         //引数にbaseAudioItemがある場合、話速等のパラメータを引き継いだAudioItemを返す
         //baseAudioItem.queryが未設定の場合は引き継がない(起動直後等？)
-        audioItem.query.speedScale = baseAudioItem.query.speedScale;
-        audioItem.query.pitchScale = baseAudioItem.query.pitchScale;
-        audioItem.query.intonationScale = baseAudioItem.query.intonationScale;
-        audioItem.query.volumeScale = baseAudioItem.query.volumeScale;
-        audioItem.query.prePhonemeLength = baseAudioItem.query.prePhonemeLength;
-        audioItem.query.postPhonemeLength =
+        newAudioItem.query.speedScale = baseAudioItem.query.speedScale;
+        newAudioItem.query.pitchScale = baseAudioItem.query.pitchScale;
+        newAudioItem.query.intonationScale =
+          baseAudioItem.query.intonationScale;
+        newAudioItem.query.volumeScale = baseAudioItem.query.volumeScale;
+        newAudioItem.query.prePhonemeLength =
+          baseAudioItem.query.prePhonemeLength;
+        newAudioItem.query.postPhonemeLength =
           baseAudioItem.query.postPhonemeLength;
-        audioItem.query.outputSamplingRate =
+        newAudioItem.query.outputSamplingRate =
           baseAudioItem.query.outputSamplingRate;
-        audioItem.query.outputStereo = baseAudioItem.query.outputStereo;
-        audioItem.morphingInfo = baseAudioItem.morphingInfo;
-      }
-
-      // audioItemに対してプリセットを適用する
-      if (shouldApplyPreset) {
-        await dispatch("APPLY_AUDIO_PRESET_TO_AUDIO_ITEM", { audioItem });
+        newAudioItem.query.outputStereo = baseAudioItem.query.outputStereo;
+        newAudioItem.morphingInfo = baseAudioItem.morphingInfo;
```

Comment:
すみません、PRの範囲と違うのですが、参照コピーになってるミスを見つけたのでもしよかったらついでに適用お願いします。。 🙇‍♂️ 
```suggestion
        newAudioItem.morphingInfo = baseAudioItem.morphingInfo
          ? { ...baseAudioItem.morphingInfo }
          : undefined;
```